frontend/src/pages/LogsPage/LogsTable/LogsTable.tsx

```diff
@@ -1,9 +1,13 @@
 import LoadingBox from '@components/LoadingBox'
 import { GetLogsQuery } from '@graph/operations'
-import { LogLine } from '@graph/schemas'
-import { Box, Stack } from '@highlight-run/ui'
-import SvgChevronDownIcon from '@icons/ChevronDownIcon'
```

Comment:
Yeah, we probably should ban the imports from `@icons` at some point. All the icons in `packages/ui` are generated from our Figma designs now, so that should be the source of truth going forward.

Reply:
Tracked in #4385

Reply:
@et thanks for creating https://github.com/highlight/highlight/issues/4385 to track this!